Web/Controllers/ConvertController.cs

```diff
@@ -5,7 +5,7 @@
 namespace ICSharpCode.CodeConverter.Web
 {
     [Route("api/[controller]")]
-    public class ConverterController : Controller
+    public class ConvertController : Controller
```

Comment:
I've changed this (and therefore the path) to be consistent with the Azure function.
If anyone is using this we could make the old path a redirect perhaps. Any thoughts?